omemo/state.py

```diff
@@ -232,7 +235,11 @@ def create_msg(self, from_jid, jid, plaintext):
         # Encrypt the message key with for each of receivers devices
         for rid, cipher in session_ciphers.items():
             try:
-                encrypted_keys[rid] = cipher.encrypt(key).serialize()
+                if self.trust[self.isTrusted(cipher)] is True:
```

Comment:
Ahh now I understand why you are doing `self.trust[self.isTrusted(cipher)] is True`.

Instead of using a dictionary for `trust` use constants:
```python
TRUST_NOT_SET  = None
TRUST_FALSE  = 0
TRUST_TRUE = 1
TRUST_UNDECIDED  = 2

if self.getTrustState(cipher) == TRUST_TRUE:
   …
```

Also you should probably replace `"Not Set"` with "`undecided`", or do you have a special purpose for the "Not Set" state?